src/vmm/src/cpu_config/x86_64/static_cpu_templates/t2s.rs

```diff
@@ -254,12 +254,13 @@ pub fn t2s() -> CustomCpuTemplate {
             // - Bit 22: Reserved
             // - Bit 23: OVERCLOCKING_STATUS
             // - Bit 24: PBRSB_NO
+            // - Bit 26: GDS_NO
             // - Bits 63-25: Reserved
             RegisterModifier {
             addr: 0x10a,
             bitmap: RegisterValueFilter {
                 filter: 0b1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111_1111,
-                value: 0b0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_0000_1000_0000_1100_0100_1100,
+                value: 0b0000_0000_0000_0000_0000_0000_0000_0000_0000_0100_0000_1000_0000_1100_0100_1100,
```

Comment:
just to clarify, we are setting the bit (not passing it through), because the guest kernel doesn't have info about the microcode?

Reply:
good question. On the host with the microcode update, GDS_NO is 0 and GDS_CTRL is 1. On the other hand, based on these values, KVM sets GDS_NO to 1 instead of passing through these bits, because guests should not have an ability to control the system-wide mitigation on the host. Firecracker follows the same policy that lets guests know they're not affected by Intel GDS rather than shows guests as if they had an ability to control the mitigation of Intel GDS.

Reply:
You can find the KVM's behavior here: https://elixir.bootlin.com/linux/latest/source/arch/x86/kvm/x86.c#L1676

Reply:
ok, that makes sense, thanks.